### ISSUE TITLE
[Epic] One Stream : One(ish) Table

### ISSUE THREAD

**Issue by jbfbell** [airbytehq/airbyte#26016]
We are changing the format of the final table(s) that Airbyte creates in the destination. The largest change is that we will no longer unnest JSON columns into their own tables. For a more thorough overview, see the [PRD](https://docs.google.com/document/d/126SLzFLMS2QYXHAItx1cn03aj0HMuuDlajlUAVFtSIM/edit#)

**Comment by jbfbell** [comment]
Relates to #24790

**Comment by jbfbell** [comment]
Update 6/6/23:
Last week we finished writing the 1s1t tech spec and are in the process of reviewing and ratifying it.

This week we identified two engineering work streams: DBT to Java translation and Spike on incremental delivery. The spike on incremental delivery should be able to make a judgement call by EOD Wednesday or Thursday as to whether the additional complexity is worth including in this phase.

We are also iterating on and reviewing the migration and rollout plan.
